ramls/rtypes/simple-crud.raml

```diff
@@ -0,0 +1,114 @@
+#%RAML 1.0 ResourceType
+# item-collection.raml
+    description: Entity representing a <<resourcePathName|!singularize>>
+    #is: [language]
```

Comment:
This is a bit inconsistent with all other endpoints...

Reply:
All comments fixed